ghostTemplate.yaml

```diff
@@ -0,0 +1,377 @@
+              Value: !Sub arn:aws:rds:${AWS::Region}:${AWS::AccountId}:cluster:${ghostDB}
+            - Name: DB_ROOT_USER_SECRET_ARN
+              Value: !Ref ghostDBRootUserSecret
+            - Name: database__connection__host
+              Value: !GetAtt ghostDB.Endpoint.Address              
+            - Name: database__connection__user
+              Value: root              
+            - Name: database__connection__password
+              Value: !Join ['', ['{{resolve:secretsmanager:', !Ref ghostDBRootUserSecret, ':SecretString:password}}' ]]
+            - Name: database__client
+              Value: mysql
+            - Name: database__pool__min
+              Value: '0'
+            - Name: database__connection__port
+              Value: '3306'
+            - Name: database__connection__database
+              Value: !Ref ghostDB
+            # string version of the Amazon Root CA 1 trust store which Ghost uses for SSL to Aurora MySQL
+            # https://www.amazontrust.com/repository/AmazonRootCA1.pem
+            # https://ghost.org/docs/concepts/config/
+            - Name: database__connection__ssl__ca
+              Value: "-----BEGIN CERTIFICATE-----\nMIIDQTCCAimgAwIBAgITBmyfz5m/jAo54vB4ikPmljZbyjANBgkqhkiG9w0BAQsF\nADA5MQswCQYDVQQGEwJVUzEPMA0GA1UEChMGQW1hem9uMRkwFwYDVQQDExBBbWF6\nb24gUm9vdCBDQSAxMB4XDTE1MDUyNjAwMDAwMFoXDTM4MDExNzAwMDAwMFowOTEL\nMAkGA1UEBhMCVVMxDzANBgNVBAoTBkFtYXpvbjEZMBcGA1UEAxMQQW1hem9uIFJv\nb3QgQ0EgMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBALJ4gHHKeNXj\nca9HgFB0fW7Y14h29Jlo91ghYPl0hAEvrAIthtOgQ3pOsqTQNroBvo3bSMgHFzZM\n9O6II8c+6zf1tRn4SWiw3te5djgdYZ6k/oI2peVKVuRF4fn9tBb6dNqcmzU5L/qw\nIFAGbHrQgLKm+a/sRxmPUDgH3KKHOVj4utWp+UhnMJbulHheb4mjUcAwhmahRWa6\nVOujw5H5SNz/0egwLX0tdHA114gk957EWW67c4cX8jJGKLhD+rcdqsq08p8kDi1L\n93FcXmn/6pUCyziKrlA4b9v7LWIbxcceVOF34GfID5yHI9Y/QCB/IIDEgEw+OyQm\njgSubJrIqg0CAwEAAaNCMEAwDwYDVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMC\nAYYwHQYDVR0OBBYEFIQYzIU07LwMlJQuCFmcx7IQTgoIMA0GCSqGSIb3DQEBCwUA\nA4IBAQCY8jdaQZChGsV2USggNiMOruYou6r4lK5IpDB/G/wkjUu0yKGX9rbxenDI\nU5PMCCjjmCXPI6T53iHTfIUJrU6adTrCC2qJeHZERxhlbI1Bjjt/msv0tadQ1wUs\nN+gDS63pYaACbvXy8MWy7Vu33PqUXHeeE6V/Uq2V8viTO96LXFvKWlJbYK8U90vv\no/ufQJVtMVT8QtPHRh8jrdkPSHCa2XV4cdFyQzR1bldZwgJcJmApzyMZFo6IQ6XU\n5MsI+yMRQ+hDKXJioaldXgjUkK642M4UwtBV8ob2xJNDd2ZhwLnoQdeXeGADbkpy\nrqXRfboQnoZsG4q5WTP468SQvvG5\n-----END CERTIFICATE-----\n"
+      Cpu: 256
+      ExecutionRoleArn: !GetAtt ghostAPIExecutionRole.Arn
+      Memory: 512
+      NetworkMode: awsvpc
+      RequiresCompatibilities:
+        - FARGATE
+      TaskRoleArn: !GetAtt ghostAPITaskRole.Arn
+      Volumes: 
+        - Name: ghostVolume
+          EFSVolumeConfiguration:
+            FilesystemId: !GetAtt ghostFS.FileSystemId
+            AuthorizationConfig:
+              IAM: "DISABLED"
+    DependsOn: ghostAPILogGroup
+  Service:
+    Type: AWS::ECS::Service
+    Properties:
+      ServiceName: !Sub ghostAPIservice-${EnvironmentTagName}
+      PlatformVersion: 1.4.0
```

Comment:
I suggest dropping this. It's unlikely future revisions will break things, and they have shipped significant improvements over time. When it's unspecified it uses the latest version by default.

Reply:
Worth removing still? I didn't see a comment reply.